strr-web/components/bcros/form-section/property/Form.vue

```diff
@@ -273,10 +273,17 @@ const validateHostResidence = () => {
 }
 
 const validateNumberOfRoomsForRent = () => {
-  if (formState.propertyDetails?.numberOfRoomsForRent < 1) {
+  let value = formState.propertyDetails?.numberOfRoomsForRent
+
+  if (!Number.isInteger(Number(value))) {
```

Comment:
What is the goal of this, check if number? Could it simplified?

Reply:
it checks if the number is an integer, as the values can be now inputs. it prevents you from adding decimals. the check whether it is a number is done via the UForm by assigning type = "number"

Reply:
Ok... and will there be an error showing?

Reply:
no, it simply doesn't let you enter decimals. restricts you to whole numbers.  for example if you enter 4.5 it will change it to 4. So you want a different behavior here?

Reply:
I'm not familiar with the requirement, so if that's what is required (to change users' input), I'm ok with it.

Reply:
Thanks for clarifying. Resolving the comment.